.golangci.yml

```diff
@@ -199,3 +199,9 @@ issues:
         - gocyclo
         - nestif
         - gosec
+    # The test is only used internally and doesn't need
+    # to meet the same standards as the rest of the codebase for now.
+    - path: run/tests/http/async/main\.go
+      linters:
+        - gosec
+      text: G114
```

Comment:
Can you add the newline? :pleading_face:

Reply:
sure...